test/integration/features/cmd-timezone.feature

```diff
@@ -0,0 +1,72 @@
+@cmd-timezone @core
+Feature: Minishift timezone subcommand
+Timezone subcommand is used for listing and applying timezone 
+availaible to Minishift running VM instance.
+
+@quick
+Scenario: User cannot run timezone subcommand without a running Minishift instance
+  Given Minishift has state "Does Not Exist"
+   When executing "minishift timezone" succeeds
+    And stdout should contain
+    """
+    Running this command requires an existing 'minishift' VM
+    """
+
+@quick
+Scenario: User cannot run timezone flag --list without a running Minishift instance
+  Given Minishift has state "Does Not Exist"
+   When executing "minishift timezone --list" succeeds
+    And stdout should contain
+    """
+    Running this command requires an existing 'minishift' VM
+    """
+
+@quick
+Scenario: User cannot set a specific timezone without a running Minishift instance
+  Given Minishift has state "Does Not Exist"
+   When executing "minishift timezone --set Asia/Kolkata" succeeds
+   Then stdout should contain
+    """
+    Running this command requires an existing 'minishift' VM
+    """
+
+Scenario: User can start Minishift with specific Asia/Kolkata timezone
+  Given Minishift has state "Does Not Exist"
+    And image caching is disabled
+   When executing "minishift start --timezone Asia/Kolkata" succeeds
+   Then executing "minishift timezone" succeeds
+    And stdout should contain
+    """
+    Asia/Kolkata
+    """
+
+Scenario: User can list out timezones associated to the running Minishift instance
```

Comment:
Maybe `User can list available timezones for the running Minishift instance` would be better.